txflood/cmd/txflood/cmd/cmd.go

```diff
@@ -0,0 +1,165 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+)
+
+var (
+	accountsFile = "accounts.json"
+	rpcURL       = "http://localhost:8545"
+)
```

Comment:
Ideally would probably turn this into a config file or something but this is fine and bare minimum works for now